### ISSUE TITLE
CellEditor width

### ISSUE THREAD

**Issue by baloniet** [avallete/ag-grid-autocomplete-editor#60]
Thank you for your excelent component, really great time saving for us!!
Is there any option to set the width of the editor. My ColumnDef has it's own width as per ag-grid documentation like this:
`...
   headerName: 'Part',
      field: 'part.name',
      width: '200',
      cellEditor: AutocompleteSelectCellEditor,
...` 
But editor itself is much shorter.
![image](https://user-images.githubusercontent.com/22891130/79007406-67af3f00-7b5b-11ea-8bfe-0159d809c37c.png)
It would be really useful.
Thank you again!!

**Comment by avallete** [comment]
Thank's for opening an issue.

This should not be the default behaviour.

Just tested it with `ag-grid-community@23.0.2` and been able to reproduce.

The bug does not exist in `ag-grid-community@22.1.1` though.

At first glance, it look like ag-grid removed some class the package used. Working on a fix and releasing ASAP.

Good catch :+1:

**Comment by avallete** [comment]
Fix has been made in the new v1.4.0 release.

This should fix the issue of the 'input' field not taking all the width of the column.

If you want also the 'select list' to take all the width (or in general to change the style of it) you can use the "customize" attribute who will look like that:
https://github.com/avallete/ag-grid-autocomplete-editor/blob/017193ba090fb0054cdae8ccd82dfdc11a148624/ag-grid-autocomplete-editor.ts#L137-L143

and will allow you to customize the rendering of the generated list of choices.

For your specific case (witch is taking the all place of the column) you can either into the 'cutomize' add this line:

```
    // By default, the list cannot take up to 25% of the width
    container.style.maxWidth = "";
```

Or to directly change the class here: https://github.com/avallete/ag-grid-autocomplete-editor/blob/017193ba090fb0054cdae8ccd82dfdc11a148624/ag-grid-autocomplete-editor.scss#L8-L17

Overriding it to remove the max-width constraint.

I close the issue. Does not hesitate to reopen if necessary.

**Comment by baloniet** [comment]
Andrew thank you this solved this issue, works as exprected. Great job!!
